views/update/review-update-statement.html

```diff
@@ -27,6 +51,10 @@
 
 {% set updateStatementDate = dateMacros.formatDate(appData.update.filing_date["day"], appData.update.filing_date["month"], appData.update.filing_date["year"]) %}
 
+{% if isRemoveJourney %}
+  {% set removeStatementDate = dateMacros.formatDate(pageParams.today["day"], pageParams.today["month"], pageParams.today["year"]) %}
```

Comment:
Maybe this does the padding and my previous question is not relevant?

Reply:
yes, the macro converts the numerics into a padded sentence - eg 05 January 2024